apps/admin/backend/src/types.ts

```diff
@@ -392,3 +395,15 @@ export const DatabaseSerializedCastVoteRecordVotesSchema: z.ZodSchema<DatabaseSe
     ContestIdSchema,
     z.union([z.array(CandidateIdSchema), YesNoVoteSchema])
   );
+
+/**
+ * @deprecated The {@link FullElectionManualTally} object is not serializable
+ * by `grout` so we have an alternative on the server side, and convert on the
+ * frontend.
+ */
+export type ServerFullElectionManualTally = Omit<
+  FullElectionManualTally,
+  'resultsByCategory'
+> & {
+  resultsByCategory: Dictionary<Dictionary<ManualTally>>;
+};
```

Comment:
We actually had the same problem on the frontend when serializing for local storage. Now we just have the problem somewhere else. This is a temporary measure while we're serving the whole full election tally to the frontend.